monash-human-power/dashboard
# Move server project files to server/

## Description

We used to share dependencies/node_modules between the server and client and this required `package.json` and other support files to be placed in the root directory. Now that the client has its own project structure, this is not required anymore, and causes the client pre-flight check to fail. This PR moves all the server's project files into the server/ directory.

## Screenshots

N/A

## Steps to Test

Remove the `node_modules` folder. Go to `server/` and run `yarn install`. All the regular server commands should work.
